rfcs/RFC-0002 - Comment Nodes For All Comments.md

```diff
@@ -0,0 +1,269 @@
+# RFC-0002 - Comment Nodes For All Comments
+
+## Prerequisites
+
+* Understand comment ownership: https://github.com/Microsoft/TypeScript/wiki/Architectural-Overview#trivia
+* Understand the difference between `getChildren()` and `forEachChild` in the compiler API.
+
+## Issues
+
```

Comment:
* [RFC-0002](RFC-0002 - Comment Nodes For All Comments.md) - Comment nodes for all comments.